### DISCUSSION TITLE
[Tests] Add test for the user model

### DISCUSSION THREAD

**Discussion by MxOliver** [Terrastories/terrastories#726] [Temporarily Archived Tickets]
Add tests for the user model (`app/models/user.rb`)

**Comment by TheIncredibleHarsh** [comment]
I'm trying this

**Comment by TheIncredibleHarsh** [comment]
I am facing problem trying to setup postgres on my macbook pro 2019 macOSX mojave

**Comment by kalimar** [comment]
@TheIncredibleHarsh moving the question over here https://github.com/Terrastories/terrastories/issues/292

You can also ask in our slack channel. Join slack teamhere https://rubyforgood.herokuapp.com/
and look for #terrastories channel

Thanks again for contributing.

**Comment by sleekybadger** [comment]
I opened https://github.com/Terrastories/terrastories/pull/308 that should close this.

**Comment by mirandawang** [comment]
@sleekybadger Added a question for you in #308 -  thank you!!
